src/Platform/Microsoft.Testing.Extensions.VSTestBridge/Helpers/TestApplicationBuilderExtensions.cs

```diff
@@ -44,4 +46,13 @@ public static void AddRunSettingsService(this ITestApplicationBuilder builder, I
     /// <param name="extension">The extension that will be used as the source of registration for this helper service.</param>
     public static void AddTestRunParametersService(this ITestApplicationBuilder builder, IExtension extension)
         => builder.CommandLine.AddProvider(() => new TestRunParametersCommandLineOptionsProvider(extension));
+
+    /// <summary>
+    /// Register the environment variable provider.
+    /// </summary>
+    /// <param name="builder">The test application builder.</param>
+    /// <param name="extension">The extension that will be used as the source of registration for this helper service.</param>
+    public static void AddEnvironmentVariableProvider(this ITestApplicationBuilder builder, IExtension extension)
```

Comment:
Maybe `AddRunSettingsEnvironmentVariableProvider` to have more specific name. Also is there a need to split the functionality? Otherwise we could fit it under the `AddRunSettingsService` on line 32.